CHANGELOG.md

```diff
@@ -9,11 +9,14 @@
   * `cortex_ingester_sent_bytes_total`
 * [CHANGE] Experimental blocks storage: the `operation` label value `getrange` has changed into `get_range` for the metrics `thanos_store_bucket_cache_operation_requests_total` and `thanos_store_bucket_cache_operation_hits_total`. #3000
 * [CHANGE] Experimental Delete Series: `/api/v1/admin/tsdb/delete_series` and `/api/v1/admin/tsdb/cancel_delete_request` purger APIs to return status code `204` instead of `200` for success. #2946
+* [CHANGE] TLS server validation is now enabled by default, a new parameter `tls_insecure_skip_verify` can be set to true to skip validation optionally. #3030
 * [ENHANCEMENT] Add support for azure storage in China, German and US Government environments. #2988
 * [ENHANCEMENT] Query-tee: added a small tolerance to floating point sample values comparison. #2994
 * [ENHANCMENT] Expose `storage.aws.dynamodb.backoff_config` configuration file field. #3026
+* [ENHANCMENT] Provide `tls_server_name` to require a specific server name instead of the hostname on the certificate. #3030
```

Comment:
Doh!  `[ENHANCMENT]` should be `[ENHANCEMENT]`. Could you fix the CHANGELOG entry 1 line above as well, please? 🙏

Reply:
Was updated by someone else meanwhile